recommender/recommender.py

```diff
@@ -1023,29 +1022,22 @@ def parse_xml(cls, node, runtime, keys, _id_generator):  # pylint: disable=unuse
 
         """
         block = runtime.construct_xblock_from_class(cls, keys)
-
-        node_xml = etree.tostring(node, encoding='utf-8')
-        try:
-            dom = md.parseString(node_xml.encode('utf-8'))
-        except Exception:
-            raise UpdateFromXmlError("An error occurred while parsing the XML content.")
-        root = dom.documentElement
-
-        if root.tagName != 'recommender':
+        if node.tag != 'recommender':
             raise UpdateFromXmlError("XML content must contain an 'recommender' root element.")
 
-        block.intro_enabled = (root.getAttribute('intro_enabled').lower().strip() != "false")
-        block.client_side_settings['DISABLE_DEV_UX'] = (root.getAttribute('DISABLE_DEV_UX').lower().strip() != "false")
+        if node.get('intro_enabled'):
+            block.intro_enabled = (node.get('intro_enabled').lower().strip() not in ['false', '0', ''])
```

Comment:
Since you can't pass booleans to the `set()` method, `lxml` probably won't parse them automatically either. You can ignore this comment.